src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -25,76 +25,97 @@
       :on-press            #(rf/dispatch [:navigate-to :wallet-address-watch])
       :add-divider?        true}]]])
 
-(def account-cards
-  [{:name                "Account 1"
-    :balance             "€0.00"
-    :percentage-value    "€0.00"
-    :customization-color :blue
-    :type                :empty
-    :emoji               "🍑"
-    :on-press            #(rf/dispatch [:navigate-to :wallet-accounts])}
-   {:customization-color :blue
-    :on-press            #(rf/dispatch
-                           [:show-bottom-sheet {:content new-account}])
-    :type                :add-account}])
+(def add-account-placeholder
+  {:customization-color :blue
+   :on-press            #(rf/dispatch [:show-bottom-sheet {:content new-account}])
+   :type                :add-account})
 
 (def tabs-data
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
+(defn get-balance
+  [balances address]
+  (->> balances
+       (filter #(= (:address %) address))
+       first
+       :balance))
+
+(defn refactor-data
+  [accounts loading? balances]
+  (let [refactored-accounts (mapv (fn [account]
+                                    (assoc account
+                                           :type                :empty
+                                           :customization-color :blue
+                                           :on-press            #(rf/dispatch [:navigate-to
+                                                                               :wallet-accounts])
+                                           :loading?            loading?
+                                           :balance             (str "$"
+                                                                     (get-balance balances
+                                                                                  (:address account)))))
+                                  accounts)]
+    (conj refactored-accounts add-account-placeholder)))
+
+(defn- view-internal
+  [accounts]
+  (let [top          (safe-area/get-top)
+        selected-tab (reagent/atom (:id (first tabs-data)))
+        loading?     (rf/sub [:wallet-2/tokens-loading?])
+        balances     (rf/sub [:wallet-2/balances])]
+    [rn/view
+     {:style {:margin-top top
+              :flex       1}}
+     [common.top-nav/view]
+     [rn/view {:style style/overview-container}
+      [quo/wallet-overview temp/wallet-overview-state]]
+     [rn/pressable
+      {:on-long-press #(rf/dispatch [:show-bottom-sheet {:content temp/wallet-temporary-navigation}])}
+      [quo/wallet-graph {:time-frame :empty}]]
+     [rn/flat-list
+      {:style      style/accounts-list
+       :data       (refactor-data accounts loading? balances)
```

Comment:
hmm, I think `refactor-data` is a bit too generic and doesn't read so well. This data is the list of cards. 
Can we call this `account-cards` or something to suggest what this data is representing

Reply:
Done